dspace-api/src/main/java/org/dspace/content/DCSeriesNumber.java

```diff
@@ -81,7 +81,7 @@ public String toString()
         {
             return (null);
         }
-        else if (number == null)
+        else if (number == null || number.replaceAll("\\s","").length() == 0)
```

Comment:
Actually, other places in DSpace use `StringUtils.isBlank(number)` from commons-lang ([javadocs](https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#isBlank(java.lang.String))) -- it would be good to do so here too for consistency. The added advantage is that it does a null check too.